src/components/Filters/index.tsx

```diff
@@ -416,6 +417,15 @@ class Filters extends React.Component<FiltersProps, FiltersState> {
 				>
 				{map(this.state.edit, ({ field, operator, value }, index) => {
 					const operators = this.getOperators(field);
+					const fieldObject = this.state.schema.properties![field];
+
+					if (value === '') {
```

Comment:
We shouldn't be updating state as part of the render method like this, Can you move this logic into `setEditField` ?

Reply:
@LucianBuzzo that won't solve the problem as it won't be called until the user triggers the `onChange` event. We need to somehow set the value as a default.

Reply:
@Shou You can put your logic in `getCleanEditModel` as that gets called on construction and when the field selector changes. I just messed around with it and this seems to work a treat
```ts
	getCleanEditModel(field?: string | null) {
		const schema = this.state.schema;
		if (!field) {
			field = Object.keys(schema.properties!).shift()!;
		}
		const operator = this.getOperators(field).shift()!.slug;
		let value: any = '';

		const subschema = schema.properties![field];
		if (typeof subschema !== 'boolean') {
			if (subschema.enum) {
				value = subschema.enum[0] || '';
			}

			if (subschema.type === 'boolean') {
				value = true;
			}
		}

		return {
			field,
			operator,
			value,
		};
	}
```

Reply:
Sorry it took so long here – yeah that works, thanks! I've updated the PR with those changes now.